### PR TITLE
Fix #7: Contextually escape data

### LINKED ISSUE #7: All values are string
All generated values in the INSERT statements have quotation marks surrounding them, and integer values that are zeros in the dbf table, are inserted as empty string which triggers an error in Mysql server

### PR DESCRIPTION
**Pull request by xmorave2**

### CODE REVIEW

**Comment by radiuscz** [comment]
Test plan...? ;-)

**Comment by xmorave2** [comment]
Test plan:
* Try to convert some dbf with integer/double/float typed columns
* Number types should not be enclosed in quotes
* Try to import that sql to the mysql/mariadb, it should work and have to import correct data (numbers - zeros not converted to empty string"

File: dbf2sql.php [hunk 1]
```diff
@@ -82,7 +82,7 @@ function usage($errormessage = "error")
             if (($column->getType() == Record::DBFFIELD_TYPE_DATETIME) && $cell) {
                 $cell = date('Y-m-d H:i:s', $cell-3600);
             }
-            $row .= "\"" . addslashes($cell) . "\",";
+            $row .= escData($cell) . ",";
```
Comment (radiuscz) [inline dbf2sql.php]: Chybí tu parametr type, který funkce escData vyžaduje pro svůj běh.

Comment (xmorave2) [reply dbf2sql.php]: Opraveno. díky

**Pull request merged**